### DISCUSSION TITLE
inQuiry Community Connect | Bootcamp Followup Session

### DISCUSSION THREAD

**Discussion by sajeshkayyath** [Sunbird-inQuiry/Community#70] [Announcements]
Hey folks! We would like to encourage you to try out other problem statement from the Sunbird Developer Bootcamp as detailed below and use the upcoming connect session to show and tell the solutions you have come up with. 

Those of you who had not worked on the front-end problem statements can also attempt this and join the upcoming session.

**_The connect is planned for 5th of April 2023 between 5 to 6 PM IST._**

Further details on the connect will be shared with you in the later updates.

Feel free to post any queries / questions you have regarding the problem statements and we can provide the necessary details.

**Problem Statement Details**

**Use case 1**: [IQ-298](https://project-sunbird.atlassian.net/browse/IQ-298) **QuML Player with configurable score and shuffle the Questions**
_The QuML player uses the Question Set and plays the questions. It generates the telemetry and computes the score. It also has the ability to shuffle the questions of the Question Set and to have a different experience for each play._
_But, it uses score as zero or “1” for all the questions. Please enhance the implementation to update the score to any defined value to compute the score._

**Use case 2**: [IQ-287](https://project-sunbird.atlassian.net/browse/IQ-287) **QuML Player – Question with audio as solution**
_The QuML Player has the capability to render solutions as only the video, text and image. Please enhance the implementation to support audio._

**Use case 3**: [IQ-295](https://project-sunbird.atlassian.net/browse/IQ-295) **Enhance QuML player to support multiples Question sets**
_The Question Set is a collection which allows multiple Questions and Question Sets in a nested graph structure._
_The QuML player has a limitation to support only one Question Set to play. Please enhance the implementation to support nested structures having multiple Question Sets._

Please refer below for more detail on inQuiry

[Microsite](https://inquiry.sunbird.org/)

[Sunbird ED Developer Bootcamp 2023 - Sunbird inQuiry.pdf](https://github.com/Sunbird-inQuiry/Community/files/11078293/Sunbird.ED.Developer.Bootcamp.2023.-.Sunbird.inQuiry.pdf)

@pramodkvarma, @vrayulu @SanthoshVasabhaktula @mathewjpallan @surendrasinghs @PoojaJManjooran @sajeshkayyath @kameshbhr @vaibhavbhuva @rajnishdargan @krgauraw @Krishnaj20 @madhucr @rhwarrier @mmanimegalai @HarishGangula
@israr-trigyn @Xitija @vishalisakar @mansurskTarento @BharathTarento @adithyadinesh0412 @shashankpace @shwetha-rao13 @Vinod-V3 @snehar-nd @panduankani @shreyasb22 @pallakartheekreddy

**Comment by sajeshkayyath** [comment]
Tagging all the participants

@libin120 @darpandeb @sudarshanswati @manthanpandey23 @manish050994 @techframewirk @bbmukherjee @vbg @dheeraj-nd @vishwab1 @JagadishPujari @vinayakmetagoli @sumanvpacewisdom @Vivek-M-08 @5Amogh @Ashwiniev95 @nevil-mathew @Shakthieshwari @suyash2000 @pathiktarento1089 @mahesh-maney @santhosh-tarento @shishirsuman092 @vky25 @karthik-tarento @SonaArun @radheshhathwarbh @Hellstellar @AmithRangadhol04 @krishnakr6 @LavanyaKR3007 @Sanjay-krishnan @Jyotiannoor  @yogeshsarmalkar-trigyn @ranvsin2 @Debayanpaul1593 @ytk1389

**Comment by PoojaJManjooran** [comment]
One more day left for the inQuiry Virtual Community Connect!

### Event Details 
Date: 5th April 2023 
Time: 5 to 6 PM IST

Zoom Link: [https://us06web.zoom.us/j/85480198431?pwd=TTdzTWNVY0IwZi9lRExYakJrMGFBUT09Meeting](https://www.google.com/url?q=https://us06web.zoom.us/j/85480198431?pwd%3DTTdzTWNVY0IwZi9lRExYakJrMGFBUT09Meeting&sa=D&source=calendar&ust=1680966042507106&usg=AOvVaw3oqSoAzO29c6QI2Zo4sFIz) 
ID: 854 8019 8431 
Passcode: 426890

<img width="313" alt="Screenshot 2023-04-03 at 11 58 02 AM" src="https://user-images.githubusercontent.com/116065864/229684752-24aa78d3-8035-483c-a77b-c624826e9c75.png">
